src/components/Restoration/RestorationType.vue

```diff
@@ -1,56 +1,63 @@
 <template>
-  <v-card flat id="restoration-type" class="pa-6">
```

Comment:
To be consistent with other components hanging off the parent page, I moved the v-card to the parent and I moved the invalid-section styling in here.

Example of WORKING validation error:

![image](https://user-images.githubusercontent.com/10002889/218227893-ff5779d6-0846-4c43-ab67-37eb8a6edb21.png)

Reply:
(I also did the same thing with the Approval Type component.)

Reply:
To be consistent with other components hanging off the parent page, I moved the v-card to the parent and I moved the invalid-section styling in here.

(I also did the same thing with the Approval Type component. @JazzarKarim )

Example of restoration validation error:

![image](https://user-images.githubusercontent.com/10002889/219828980-4dd47fa2-d389-4e5f-8151-6ad5e14d2d46.png)

Example of approval type validation error:

![image](https://user-images.githubusercontent.com/10002889/219829016-2b803f30-d925-473d-b942-5bcf0e70bbc7.png)